src/components/layout/Layout.js

```diff
@@ -3,16 +3,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import GlobalStyles from '../../styles/globalStyles';
 import Header from '../header';
+import Footer from '../footer';
 
 const Layout = ({ children }) => (
   <div>
     <GlobalStyles />
     <Header />
     {children}
+    <Footer />
```

Comment:
One last thing I'd like to see: It would be great if the app would take at least the complete window height. You can wrap a component around the `children` above and give it `min-height: calc(100% - headerHeight - footerHeight)`

Reply:
done :D